Configuration/PyReleaseValidation/README.md

```diff
@@ -31,6 +31,14 @@ The offsets currently in use are:
 * 0.3: 0.1 + 0.2
 * 0.4: LowPU tracking era, `Run2_2017_trackingLowPU`
 * 0.5: Pixel tracking only + 0.1
+* 0.501: Patatrack, pixel only quadruplets, on CPU
```

Comment:
it looks like this does not belong to this PR.
Please rebase to get rid of unrelated commits.